### DISCUSSION TITLE
Creating new directory in root

### DISCUSSION THREAD

**Discussion by MrGiga** [npalm/terraform-aws-gitlab-runner#741] [General]
I am trying to create a new folder in root on my runners (not the agent its self)? 

I tried adding "mkdir /folder" to runners_userdata, but that doesn't seem to work. Looking for any suggestions.

Thanks!

**Comment by kayman-mk** [comment]
Could you please check if `/etc/gitlab-runner/runners_userdata.sh` is present on your agent and that it contains the script you expect to be executed on the runners?

How did you verify that the folder is present? Your build jobs won't see it as they are running in a docker container.

**Comment by MrGiga** [comment]
The file contains `mkdir /folder` and I verify that it is not running by logging into the runners and running `ls -la /` which returns a list without the folder

**Comment by kayman-mk** [comment]
All scripts I know starts with a shebang line. Could this be a problem? There should also be a `/var/log/cloud*` file, containing the logs of the user data script. Did you check it?
